server/trillian_log_server/main.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/extension"
 	_ "github.com/google/trillian/merkle/objhasher" // Load hashers
```

Comment:
I don't think you need the comments on these imports + in other files. The '_' implies it's being imported for side effects.